### PR TITLE
docs(README): update project structure section

### PR DESCRIPTION
**Pull request by raducristianpopa**
## Changes proposed in this pull request

We do not have the `static` folder anymore.

### CODE REVIEW

**Comment by raducristianpopa** [comment]
<h2>Extension builds preview</h2>

<table role="table">
<thead>
  <tr>
    <th>Name</th>
    <th>Link</th>
  </tr>
</thead>
<tbody>
  <tr>
    <td align="center">Latest commit</td>
    <td align="center">c2d92edda693ab024ed519cca028384213aa3c67</td>
  </tr>
  <tr>
    <td align="center">Latest job logs</td>
    <td align="center"><a href="https://github.com/interledger/web-monetization-extension/actions/runs/9834300599">Run #9834300599</a></td>
  </tr>
  <tr><td align="center">Badge</td><td align="center"><a href="https://github.com/interledger/web-monetization-extension/suites/25735521262/artifacts/1676323282">Download</a></td></tr><tr><td align="center">Badge</td><td align="center"><a href="https://github.com/interledger/web-monetization-extension/suites/25735521262/artifacts/1676323283">Download</a></td></tr>
</tbody>
</table>

**Comment by sidvishnoi** [comment]
Maybe add polyfill mention in content one?

File: README.md [hunk 1]
```diff
@@ -90,7 +90,6 @@ Inside this project, you'll see the following folders and files:
 │   ├── assets/ # Images for the extension (icon, etc.)
 │   ├── background/ # Source code for the background script
```
Comment (sidvishnoi) [inline README.md:91]:
```suggestion
│   ├── background/ # Source code for the background script/service worker
```

File: README.md [hunk 2]
```diff
@@ -90,7 +90,6 @@ Inside this project, you'll see the following folders and files:
...
 │   ├── content/ # Source  code for the content script
-│   │   ├── static/ # WM polyfill
 │   ├── popup/ # Source code for the popup UI
 │   └── shared/ # Shared utilities
```
Comment (sidvishnoi) [inline README.md:94]: Add manifest.json here? Mentioning it gets processed by webpack depending on browser (not copied as is)

**Comment by raducristianpopa** [comment]
> Maybe add polyfill mention in content one?

It was previously mentioned because we had "two" content scripts. Since everything is now running in a single content script, not really needed.

**Review by sidvishnoi — APPROVED** [review]

**Pull request merged**